index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/9afaa7c9c3.js" crossorigin="anonymous"></script>
+    <title>Profile Card</title>
+</head>
+<body>
+    <div class="card-container">
+        <div class="left-card">
+            <img src="./images/Profile Photo.JPG" alt="Photo">
+            <h1>Omotoso Olatomiwa</h1>
+            <h2>Web Developer</h2>
+            <p>I develop websites using Bootstrap and JavaScript. Currently playing around design with Figma.</p>
+            <button class="left-btn">View Profile</button>
+            <div class="icons">
+                <a href="https://twitter.com/tomiwathedaddy"><i class="fa-brands fa-twitter"></i></a>
+                <a href="https://instagram.com/tomii.wa"><i class="fa-brands fa-instagram"></i></a>
+                <a href="https://linkedin.com/in/omotoso-olatomiwa-a29359238"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="https://github.com/tomiwathedaddy"><i class="fa-brands fa-github"></i></a>
+                <a href="https://snapchat.com/tomiwathedaddy"><i class="fa-brands fa-snapchat"></i></a>
+                <a href="tommyomotouch@gmail.com"><i class="fa-solid fa-envelope"></i></a>
+            </div>
+        </div>
+        <div class="right-card">
+            <img src="./images/Profile Photo.JPG" alt="Photo">
+            <h1>Omotoso Olatomiwa</h1>
+            <h2>Web Developer</h2>
+            <p>I develop websites using Bootstrap and JavaScript. Currently playing around design with Figma.</p>
+            <button class="right-btn">View Profile</button>
+            <div class="icons">
+                <a href="https://twitter.com/tomiwathedaddy"><i class="fa-brands fa-twitter"></i></a>
+                <a href="https://instagram.com/tomii.wa"><i class="fa-brands fa-instagram"></i></a>
+                <a href="https://linkedin.com/in/omotoso-olatomiwa-a29359238"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="https://github.com/tomiwathedaddy"><i class="fa-brands fa-github"></i></a>
+                <a href="https://snapchat.com/tomiwathedaddy"><i class="fa-brands fa-snapchat"></i></a>
+                <a href="tommyomotouch@gmail.com"><i class="fa-solid fa-envelope"></i></a>
```

Comment:
The email is not properly structured in the href
See what it should look like
`<a href = "mailto: abc@example.com">Send Email</a>
`

Reply:
![fix2](https://user-images.githubusercontent.com/105456361/217334700-14f0ac72-f9d3-41a7-adde-9a53260dd67c.png)